### ISSUE TITLE
Release into ROS 2?

### ISSUE THREAD

**Issue by clalancette** [ROBOTIS-GIT/dynamixel-workbench-msgs#23]
@rjshim @routiful @robotpilot What do you think about releasing the ros2 branch into Dashing?  It would make it a little easier to build some dynamixel stuff in ROS 2.  Thanks!

**Comment by robotpilot** [comment]
@clalancette 
Yes. We think so too. We are porting existing Dynamixel, TurtleBot3, OpenMANIPULATOR-X and PRO to support ROS 2. We will proceed sequentially. :) Thank you. :+1:

**Comment by clalancette** [comment]
> Yes. We think so too. We are porting existing Dynamixel, TurtleBot3, OpenMANIPULATOR-X and PRO to support ROS 2. We will proceed sequentially. :) Thank you. +1

Great, thanks.

**Comment by clalancette** [comment]
Any update on this?  Can we release this into Rolling, Humble, Galactic, and Foxy now?  Other packages downstream would like to depend on it.

**Comment by ROBOTIS-Will** [comment]
@clalancette
Thanks for resolving the conflicts on my recent [PRs](https://github.com/ros/rosdistro/pulls?q=is%3Apr+dynamixel_workbench)!
I'll keep track on the rest of the release status.
Thanks!

**Comment by yun-goon** [comment]
It looks like the issue has been resolved, so I will close it now.
